core/invoker/src/main/scala/whisk/core/containerpool/ContainerPool.scala

```diff
@@ -197,11 +230,9 @@ class ContainerPool(childFactory: ActorRefFactory => ActorRef,
     // Container got removed
     case ContainerRemoved =>
       freePool = freePool - sender()
-      busyPool.get(sender()).foreach { _ =>
```

Comment:
This change does not looks like required for concurrency support. So better to retain the existing approach of sending message only if `busyPool` had `sender`